### PR TITLE
Fail process when deployment fails

### LINKED ISSUE #5: Create environment should throw error when environment fails
https://github.com/env0/env0-client-integrations/blob/6ad3598bd4491a989137768d33868d147e4f7786/node/env0-deploy-flow.js#L26

We currently poll for the environment to not-be "in-progress", but we do not stop the flow if it's "failed"

### PR DESCRIPTION
**Pull request by avnerenv0**
- Added Eslint
- Added Jest
- Fail when deploy doesn't finish in ACTIVE status

Closes #5

### CODE REVIEW

**Comment by avnerenv0** [comment]
https://trello.com/c/2OfgdW2Y/845-cli-deploy-does-not-inform-of-deploy-failure

File: node/env0-deploy-flow.js [hunk 1]
```diff
@@ -25,7 +25,10 @@ const createAndDeploy = async (options, environmentVariables) => {
   await setConfigurationFromOptions(environmentVariables, environment, options.blueprintId);
   await deployUtils.pollEnvironmentStatus(environment.id);
   await deployUtils.deployEnvironment(environment, options.revision, options.blueprintId);
-  await deployUtils.pollEnvironmentStatus(environment.id);
+  const lastStatus = await deployUtils.pollEnvironmentStatus(environment.id);
+  if (lastStatus != 'ACTIVE') {
```
Comment (onhate) [inline node/env0-deploy-flow.js]: should it fail on waiting for approval status?

Comment (omryhay1) [reply node/env0-deploy-flow.js]: @onhate - great catch, it shouldn't fail on that.

Comment (avnerenv0) [reply node/env0-deploy-flow.js]: https://github.com/env0/env0-client-integrations/issues/8

**Pull request merged**